src/logger.ts

```diff
@@ -0,0 +1,31 @@
+import log from 'electron-log/main'
+import { spawn } from 'node:child_process'
+import path from 'node:path'
+
+export function initializeLogger() {
+  // allow logs to be triggered from the renderer process
+  // @see https://github.com/megahertz/electron-log/blob/master/docs/initialize.md
+  log.initialize()
+
+  // log electron core events
+  // @see https://github.com/megahertz/electron-log/blob/master/docs/events.md
```

Comment:
minor nit: I think the `@see` are redundant and probably the link alone would suffice 🤔